badass.yml

```diff
@@ -172,7 +172,7 @@ jobs:
                   python_path: <<parameters.python_path>>
             - save_cache:
                   paths:
-                      - .local/share/virtualenvs
+                      - ~/.local/share/virtualenvs
```

Comment:
Does the tilde work as expected here?

Reply:
seems to here:
https://app.circleci.com/pipelines/github/arrai-innovations/sems/2049/workflows/77d996a5-d242-42f9-9c82-4fced6b78581/jobs/7677/parallel-runs/0/steps/0-109

Reply:
bonus